github-for-jira.sd.yml

```diff
@@ -42,13 +42,11 @@ config:
   environmentVariables:
     NODE_ENV: production
     NODE_OPTIONS: "--no-deprecation"
-    PORT: "8080"
-
-scaling:
-  min: 1
-  max: 1
-  instanceCount: 1
-  instance: t3.small
+    LOG_LEVEL: debug
```

Comment:
Should we use DEBUG log level by default? 

Let's put INFO as default and DEBUG as ddev override